dbsc/config/snakebids.yml

```diff
@@ -128,7 +128,7 @@ parse_args:
   --lmax:
     help: '(Mrtrix3) the maximum spherical harmonic order for the output
           FOD(s). For algorithms with multiple outputs, this should be
-          provided as a comma-sperated list of integers, one for each output
+          provided as a space-sperated list of integers, one for each output
```

Comment:
Unrelated to the changes at hand but while I'm looking at this, if `--lmax` can be ommitted, `nargs` should be `"*"`.